Apps/Inc/Tasks.h

```diff
@@ -133,5 +134,7 @@ extern OS_SEM SendCarCAN_Sem4;
  * Global Variables
  */
 // TODO: Put all global state variables here
+int8_t brake_percentage;
+int8_t accelerator_percentage;
```

Comment:
Variables should not be declared in the .h file. It will inevitably lead to a multiply defined symbol error down the line.

Also, do these really need to be global? Or can we get away with an getter function?

Reply:
Getter functions sound better indeed

Reply:
This structure was copied over from what BPS did with state variables that needed to be accessed by multiple threads but only modified by one. Putting getter functions in a file that really just defines the task thread seems a little excessive to me. We could avoid potential multiply defined symbol errors by creating some sort of car state struct to contain all this information. Just my opinion on this

Reply:
A struct with the state of the pedals was added in the latest push. There are other two structs that store states, one for the lights and another one for the switches. Hence, we could maybe have a general state struct that will store these three child structs: pedals, lights, and switches, and whichever more state structs we keep adding along the way/?